.test-infra/jenkins/job_PreCommit_PythonDocs.groovy

```diff
@@ -26,7 +26,8 @@ PrecommitJobBuilder builder = new PrecommitJobBuilder(
     timeoutMins: 30,
     triggerPathPatterns: [
       '^sdks/python/.*$',
-    ]
+    ],
+    commitTriggering: false
```

Comment:
Can we just remove these entirely? We've been a good state for a while, and I don't think keeping them is getting us much. We can always revert if there's a problem (which is what we'd need to do anyways)

Reply:
Sounds good, done

note that left jenkins/README.md as is for now